ui.apps/src/main/content/jcr_root/apps/styla/components/structure/emptypage/custommetatags.html

```diff
@@ -1,15 +1,15 @@
-<sly data-sly-use.model="de.neofonie.styla.core.controller.StylaSEO" />
+<link data-sly-test.stylaCanonical="${properties.stylaCanonical}" rel="canonical" href="${stylaCanonical}" />
+<meta data-sly-test.stylaRobots="${properties.stylaRobots}" name="robots" content="${stylaRobots}" />
+<meta data-sly-test.stylaOgTitle="${properties.stylaOgTitle}" property="og:title" content="${stylaOgTitle}" />
+<meta data-sly-test.stylaOgType="${properties.stylaOgType}" property="og:type" content="${stylaOgType}" />
+<meta data-sly-test.stylaOgUrl="${properties.stylaOgUrl}" property="og:url" content="${stylaOgUrl}" />
+<meta data-sly-test.stylaOgImage="${properties.stylaOgImage}" property="og:image" content="${stylaOgImage}" />
+<meta data-sly-test.stylaOgImageWidth="${properties.stylaOgImageWidth}" property="og:image:width" content="${stylaOgImageWidth}" />
+<meta data-sly-test.stylaOgImageHeight="${properties.stylaOgImageHeight}" property="og:image:height" content="${stylaOgImageHeight}" />
+<meta data-sly-test.stylaTwitterTitle="${properties.stylaTwitterTitle}" property="twitter:title" content="${stylaTwitterTitle}" />
+<meta data-sly-test.stylaTwitterDescription="${properties.stylaTwitterDescription}" property="twitter:description" content="${stylaTwitterDescription}" />
+<meta data-sly-test.stylaTwitterImage="${properties.stylaTwitterImage}" property="twitter:image" content="${stylaTwitterImage}" />
 
-    <meta data-sly-test.ogTitle="${properties.ogTitle}" property="og:title" content="${ogTitle}" />
-    <meta data-sly-test.ogType="${properties.ogType}" property="og:type" content="${ogType}" />
-    <meta data-sly-test.ogUrl="${properties.ogUrl}" property="og:url" content="${ogUrl}" />
-    <meta data-sly-test.ogImage="${properties.ogImage}" property="og:image" content="${ogImage}" />
-
-<sly data-sly-list.metaTags="${model.getMetaTags}">
-    <sly data-sly-test.isProperty="${metaTags.isProperty}">
-        <meta data-sly-test.tagLabel="${metaTags.getTagLabel}" data-sly-test.tagValue="${metaTags.getTagValue}" property="${tagLabel}" content="${tagValue}" />
-    </sly>
-    <sly data-sly-test.isProperty="${!isProperty}">
-        <meta data-sly-test.tagLabel="${metaTags.getTagLabel}" data-sly-test.tagValue="${metaTags.getTagValue}" name="${tagLabel}" content="${tagValue}" />
-    </sly>
-</sly>
+<link rel="dns-prefetch" href="//client-scripts.styla.com" />
+<link rel="dns-prefetch" href="//redpanda.styla.com" />

```

Comment:
Should we switch from client-script to engine.styla.com? and does it make sense to remove redpanda.styla.com since it's only required for the magazine feed?